src/models.ts

```diff
@@ -47,3 +47,23 @@ export class City extends Country {
     this.subdivisions = response.subdivisions || [];
   }
 }
+
+/** Class representing the model of a "ASN" response **/
+export class Asn {
+  public readonly autonomous_system_number: number | undefined;
+  public readonly autonomous_system_organization: string | undefined;
+  public ip_address: string | undefined;
```

Comment:
This `undefined` can be removed too

Reply:
I'm not sure it can due to https://github.com/runk/node-maxmind/blob/master/index.d.ts#L129

Reply:
Ah true, it can't.  Thanks for checking.

Lines 53-55 can use `prop?: type` instead of `prop: type | undefined`

Reply:
Ah yes, updated in 1ee1d6a